api/hackerProfile.js

```diff
@@ -340,4 +340,9 @@ router.post("/undecline", async (req, res) => {
   });
 });
 
+router.get("/list", async (req, res) => {
```

Comment:
iirc, this can be accessed by anyone including non-admins -- recommend adding some sort of admin check

Reply:
actually to simplify this, just add a env check to see if we're in a dev environment